src/components/ConferencePanel/styles.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../lib/commonStyles/colors';
+
+.showConferenceNumbers {
+  margin:10px 15px 0 15px;
+  height:20px;
+  transition: height 0.5s ease-in;
+  overflow:hidden;
+  font-size:13px;
+  label{
+      float:left;
+      color: #b3b3b3;
+      line-height:20px;
+      display:block;
+      width:60px;
+  }
+  .dialInNumber{
+      float:right;
+      margin-left:10px;
+      line-height:20px;
+      color:#0073ae;
+  }
+  .conferenceNumber{
+      float:right;
+      margin-left:10px;
+      line-height:20px;
+      color:#2f2f2f; 
+  }
+}
+.participantsSwitch{
+    margin:10px 15px 0 15px;
```

Comment:
indent count is different in this file. It will be better if indent is all 2 space